### PR TITLE
librtlsdr: remove

### PR DESCRIPTION
**Pull request by Tungsten842**
###### Description of changes
The same package was packaged twice, so removing one of them.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -813,6 +813,7 @@ mapAliases ({
   libressl_3_2 = throw "'libressl_3_2' has reached end-of-life "; # Added 2022-03-19
   librevisa = throw "librevisa has been removed because its website and source have disappeared upstream"; # Added 2022-09-23
   librsync_0_9 = throw "librsync_0_9 has been removed"; # Added 2021-07-24
+  librtlsdr = throw "librtlsdr is now called rtl-sdr";
```
Comment (kilianar) [inline pkgs/top-level/aliases.nix:816]: The EditorConfig check is failing due to a trailing whitespace.

**Review by bjornfor — COMMENTED** [review]
Can you please write in the commit message about this only removing a duplicate package? I wast at first afraid the package was dead upstream.

Are the Nix exprs similar enough that we can use a backward compat alias instead of a throw?

**Comment by QuantMint** [comment]
> Can you please write in the commit message about this only removing a duplicate package? I wast at first afraid the package was dead upstream.
> 
> Are the Nix exprs similar enough that we can use a backward compat alias instead of a throw?

Done

**Review by bjornfor — APPROVED** [review]

**Comment by dotlambda** [comment]
This breaks eval when aliases are not allowed:
```
error: Function called without required argument "librtlsdr" at pkgs/applications/radio/guglielmo/default.nix:3
```

**Pull request merged**